### PR TITLE
Added Subversive 3.0+ m2e connector

### PR DESCRIPTION
**Pull request by bindul**
Added Subversive connector (org.eclipse.team.svn.m2e.feature.group) for newer v3.0+ releases of Subversive which comes with its own M2E connector (see http://dev.eclipse.org/svnroot/technology/org.eclipse.subversive/trunk/org.eclipse.team.svn.m2e-feature/).

Discussion on mailing list: http://dev.eclipse.org/mhonarc/lists//m2e-users/msg05181.html

- Bindul

### CODE REVIEW

File: org.eclipse.m2e.discovery.oss/src/main/resources-filtered/connectors.xml [hunk 1]
```diff
@@ -282,6 +282,26 @@ Finally, m2e-wtp helps you convert your legacy Eclipse projects to Maven, by tra
...
     </catalogItem>
+    
+    <catalogItem>
+        <categoryId>org.eclipse.m2e.discovery.category.scm</categoryId>
+        <description>Subversive (3.0+) SVN Integration for the M2E Project</description>
+        <m2e-versions>1.6</m2e-versions>
+        <groupId>extras</groupId>
+        <id>org.eclipse.m2e.discovery.lifecyclemapping.m2e-subversive</id>
+        <kind>extras</kind>
+        <license>Eclipse Public License - v1.0</license>
```
Comment (fbricon) [inline org.eclipse.m2e.discovery.oss/src/main/resources-filtered/connectors.xml]: `<license>EPL</license>`

To be consistent with other entries

File: org.eclipse.m2e.discovery.oss/src/main/resources-filtered/connectors.xml [hunk 2]
```diff
@@ -282,6 +282,26 @@ Finally, m2e-wtp helps you convert your legacy Eclipse projects to Maven, by tra
...
+        <name>m2e-subversive</name>
+        <provider>Eclipse.org</provider>
+        <p2>
+            <repositoryUrl>http://download.eclipse.org/technology/subversive/3.0/update-site/</repositoryUrl>
+            <iuId>org.eclipse.team.svn.m2e.feature.group</iuId>
+      </p2>
+      <overview>
+        <summary>Integration with M2E http://www.eclipse.org/m2e Installation is possible only if M2E is already installed or installed in one step with Eclipse Subversive.</summary>
```
Comment (fbricon) [inline org.eclipse.m2e.discovery.oss/src/main/resources-filtered/connectors.xml]: This summary is only relevant when installing manually from the subversive update site.

Use `<summary>Maven SCM Handler for Subversive</summary>` to be consistent with the other subversive entry

**Comment by bindul** [comment]
@fbricon I have updated the pull request with the changes noted.

**Comment by andham** [comment]
I suggest changing the summary to Maven SCM Handler for Subversive 3.0 as it is only for Subversive 3.0. There is a different connector for Subversive 2.0 for example.

**Comment by fbricon** [comment]
I fixed the summary in https://github.com/takari/m2e-discovery-catalog/commit/9642f331fe519338501696c2533e9a8c18626571

**Pull request merged**